ocrolib/common.py

```diff
@@ -684,13 +680,28 @@ def ocropus_find_file(fname,gz=1):
     if gz:
         if os.path.exists(fname+".gz"):
             return fname+".gz"
-    for path in data_paths:
-        full = path+"/"+fname
+
+    for path in _data_paths:
+        full = os.path.join(path, fname)
         if os.path.exists(full): return full
     if gz:
-        for path in data_paths:
-            full = path+"/"+fname+".gz"
+        for path in _data_paths:
+            full = os.path.join(path, fname + ".gz")
             if os.path.exists(full): return full
+
+    share_path = os.path.dirname(inspect.getfile(inspect.currentframe()))
+    share_path = os.path.join(share_path,
```

Comment:
I think this maybe can be made a little nicer using `sysconfig`. Should I PR you or just open a new one here?